File: Python/account.py
class Account:
    id = int
    name = str
    document = str
    email = str
    password = str
    
    def __init__(self,id,name,document,password):
        self.id = id
        self.name=name
        self.password=password
        self.document=document
    
    def printAccountData(self):
        print("Id conductor: " + str(self.id) + 
        "nombre: " + self.name + 
        "documentación " + self.document)

File: php/Account.php
<?php
class Account {
public int $id ;
public String $name;
public String $document;
public String $email;
public String $password;
    function __construct($id,$name,$document,$email,$password) {
        $this->id=$id;
        $this->email=$email;
        $this->password=$password;
        $this->name = $name;
        $this->document = $document;
     }
}